src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const swapped = {};
+
+  for (const key in robot) {
+    if (swapped.hasOwnProperty(robot[key])) {
+      return null;
+    } else {
+      swapped[robot[key]] = key;
+    }
```

Comment:
when you have return inside if statement - no needs to use else statement
```suggestion
    } 
    
    swapped[robot[key]] = key;
    
```